src/seedu/addressbook/data/AddressBook.java

```diff
@@ -24,13 +25,15 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
+    public ArrayList<Tagging> tagging; //Arraylist of tagging records
```

Comment:
Good, this is a correct place to put the list based on the UML.. `taggingList` might be a better name. Then, you can remove the comment.